autoscaler/controllers/actions/probabilisticsampler_controller.go

```diff
@@ -1,12 +1,9 @@
 /*
```

Comment:
The probabilistic sampler should be applied in cluster collector and not node collector, so it will not filter out long latency traces which user want to keep